.github/workflows/ci.yml

```
@@ -90,16 +81,16 @@ jobs:
       - name: 🔬 Format
         run: pnpm format:fix
 
+      - name: 📥 Pull latest changes
+        run: git pull --rebase origin ${{ github.head_ref }}
```

actionlint (1.7.7)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)